tests/rt.sh

```diff
@@ -421,6 +421,7 @@ else
 fi
 
 INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20210614}
+INPUTDATA_ROOT_GOCART=${INPUTDATA_ROOT_GOCART:-$DISKNM/NEMSfv3gfs/input-data-20210614/GOCART}
```

Comment:
Is this variable needed? Can you point to gocart data using `${INPUTDATA_ROOT}/GOCART`?

Reply:
Commit 53041b2f8d0b73f942431cbaeb7619608ed207fb removed `INPUTDATA_ROOT_GOCART`